.github/policies/resourceManagement.yml

```diff
@@ -709,6 +733,8 @@ configuration:
           label: area-infrastructure
       - addLabel:
           label: 'Type: Dependency Update :arrow_up_small:'
+      - addLabel:
+          label: 'build-ops'

```

Comment:
I still see several uses like right above this one for the `Type: Dependency Update...` labels. Do we still need them or is it okay to just use build-ops label going forward? And if so, can we remove the old labels?

Reply:
fair enough, just wanted to check in case they weren't needed any longer.